app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -10,7 +10,20 @@ def google_oauth2
                               kind: t('omniauth.google'))
       sign_in_and_redirect @user, event: :authentication
     else
-      redirect_to new_user_session_path, notice: t('omniauth.access_denied')
+      redirect_to new_user_session_path, alert: t('omniauth.access_denied')
+    end
+  end
+
+  def github
```

Comment:
I am just curious why we have github auth, are our users mostly from github?

Reply:
this is something that @julianguyen can answer, but if I may try. since this is a platform that encourages collaboration, maybe a target audience of tech people are in mind?

Reply:
Yeah GitHub authentication isn't a priority like Facebook would be for instance. This was requested by a contributor 🙃

But we could use this functionality to show links for filing GitHub issues just for folks signed in it.

Reply:
We should also test this new method!